tests/components/webostv/test_config_flow.py

```diff
@@ -153,26 +154,23 @@ async def test_form(hass, client):
     "apps, inputs",
     [
         # Live TV in apps (default)
-        (None, None),
+        (MOCK_APPS, MOCK_INPUTS),
```

Comment:
The constants are not modified by the tests, right?

Reply:
No they are not modified